File: CPP/LCM.c
#include<stdio.h>
int main()
{
    int a,b,lcm,ref=2;
    scanf("%d%d",&a,&b);
    lcm = (a>b)?a:b;
    while(1)
    {
        if(lcm%a==0 && lcm%b==0)
        {
            printf("%d",lcm);
            break;
        }
        else
        {
            lcm *= ref;
            ref++;
        }    
    }
    return 0;
}

File: Python/LCM.py
ref = list(map(int,input().split()))
a = ref[0]
b = ref[1]
lcm = max(a,b)
while True:
    if(lcm%a==0 and lcm%b==0):
        print("LCM is :",lcm)
        break
    else:
        lcm = lcm+1